utils/rawblkdev.c

```diff
@@ -125,10 +206,118 @@ static int _get(myst_blkdev_t* dev, uint64_t blkno, void* data)
         }
     }
 
+#ifdef USE_LRU
```

Comment:
LRU can be organized as an array of most recently used block lists indexed by `blkno % LRU_SIZE`. This way the cache lookup can be as fast as O(1) (if there is just one cached block in that list).

Reply:
It helps some: **43,133,979** instructions vs **44,309,715** instructions in the ``bigfile`` example.

This will never be used in production since the (veritybldev.c is used in production).

Reply:
Nice idea about the optimization. I will consider but rawblkdev is never used in production so probably not a priority.

Reply:
I added locking to the free lists in rawblkdev (had forgotten before).

Reply:
I implemented your optimization. The ``bigfile`` example run time decreased from **43,274** instructions to **39,501** instructions or **8.6%**.